src/components/atom/AppCalendar.tsx

```diff
@@ -0,0 +1,52 @@
+import { Calendar, CalendarChangeParams, CalendarMonthNavigatorTemplateParams } from "primereact/calendar"
+import { useTranslation } from "react-i18next";
+import { namespaces } from "../../i18n/i18n.constants";
+import { Dropdown, DropdownChangeParams } from "primereact/dropdown";
+import 'primereact/resources/primereact.css';
+import 'primeicons/primeicons.css';
+import { current } from "@reduxjs/toolkit";
+
+interface AppCalendarProps {
+    id?: string
+    placeholder?: string,
+    selectedDate?: Date,
+    setDate: (date: string) => void
+    yearRange?: string
+}
+
+const currentYear = new Date().getFullYear()
+const MIN_BIRTH_YEAR = currentYear - 120
```

Comment:
I would try to avoid magic number like `120`. Why that number? if we use it we must be able to explain in a variable name